#request: servo deegrees to scan
float32 servo_start
float32 servo_end
---
#result: all reads of the ultrasonic + each servo positivo for each read
float64[] ultrasonic_reads
float64[] servo_deegres
---
#constant feedback: actual ultrasonic read + actual servo position
float64 ultrasonic_read
float64 servo_degree